### PR TITLE
Bugfix #2058 loqusdb feature 200901

### LINKED ISSUE #2058: ConfigError: LoqusDB id not found
While testing some other branches on stage we started getting the following error:

```
File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/utils.py", line 23, in decorated_function
    context = func(*args, **kwargs)
  File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/blueprints/variant/views.py", line 48, in variant
    data["observations"] = observations(store, loqusdb, data["case"], data["variant"])
  File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/blueprints/variant/controllers.py", line 267, in observations
    obs_data = loqusdb.get_variant(variant_query, loqusdb_id=institute_obj.get("loqusdb_id")) or {}
  File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/extensions/loqus_extension.py", line 130, in get_variant
    cmd = self.get_command(loqusdb_id)
  File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/extensions/loqus_extension.py", line 280, in get_command
    path = [self.get_bin_path(loqusdb_id)]
  File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/extensions/loqus_extension.py", line 197, in get_bin_path
    raise ConfigError("LoqusDB id not found")
scout.exceptions.config.ConfigError: LoqusDB id not found
```

We can't deploy a new release if we don't fix this first. Could you take a look at this @mikaell , might be related with you recent PR.

### PR DESCRIPTION
**Pull request by mikaell**
This PR fixes a bug. fix #2058

**How to test**:
1. Install branch.
2. In Web-GUI go to Institute settings `http://localhost:5000/overview/cust000/settings?`
3. Delete "LoqusDB id" and save
4. Go to a Variant. This should not crash.

**Expected outcome**:
The functionality should be working. No crash.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Review by northwestwitch — CHANGES_REQUESTED** [review]
Tried it:
- **Locally** - and didn't really test it according to the description because I don't have any loqusdb instance running so I couldn't remove it from the settings. Local instance works! 🆗 
- **On stage** - Looks like there is some problem in the code, check this variant for instance: https://scout-stage.scilifelab.se/cust000/15026-mip7/ac1be94991570e313ecc6146b24462e0, that produces the following error:
```
Traceback (most recent call last):
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/utils.py", line 23, in decorated_function
    context = func(*args, **kwargs)
  File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/blueprints/variant/views.py", line 48, in variant
    data["observations"] = observations(store, loqusdb, data["case"], data["variant"])
  File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/blueprints/variant/controllers.py", line 267, in observations
    obs_data = loqusdb.get_variant(variant_query, loqusdb_id=institute_obj.get("loqusdb_id")) or {}
  File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/extensions/loqus_extension.py", line 155, in get_variant
    output = execute_command(cmd)
  File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/extensions/loqus_extension.py", line 33, in execute_command
    cmd_string = " ".join(cmd)  # add spacing to create a command string
TypeError: sequence item 19: expected str instance, list found
```
This error does't occur with all variants, but I don't know what triggers it.

Comments: perhaps accept the suggestion of Flake8 and do the comparison of using == instead of with "is"?

**Comment by mikaell** [comment]
Good find. I didn't hit those triggering variants myself at first.

Now patched. Please try again!

File: scout/server/extensions/loqus_extension.py [hunk 1]
```diff
@@ -138,15 +138,15 @@ def get_variant(self, variant_info, loqusdb_id=None):
                     "sv",
                     "-c",
                     variant_info["chrom"],
-                    "-s",
+p                    "-s",
```
Comment (northwestwitch) [inline scout/server/extensions/loqus_extension.py:141]: ?

File: scout/server/extensions/loqus_extension.py [hunk 2]
```diff
@@ -188,13 +188,12 @@ def get_bin_path(self, loqusdb_id=None):
         Returns:
             path_to_bin(str)
         """
-        if loqusdb_id is None:
+        if loqusdb_id is None or loqusdb_id is "":
```
Comment (northwestwitch) [inline scout/server/extensions/loqus_extension.py:191]:
```suggestion
       if loqusdb_id is None or loqusdb_id is "":
```

File: scout/server/extensions/loqus_extension.py [hunk 3]
```diff
@@ -205,13 +204,13 @@ def get_config_path(self, loqusdb_id=None):
         Returns:
             path_to_cfg(str)
         """
-        if loqusdb_id is None:
+        if loqusdb_id is None or loqusdb_id is "":
```
Comment (northwestwitch) [inline scout/server/extensions/loqus_extension.py:207]:
```suggestion
        if loqusdb_id is None or loqusdb_id == "":
```

File: scout/server/extensions/loqus_extension.py [hunk 4]
```diff
@@ -221,13 +220,13 @@ def get_configured_version(self, loqusdb_id=None):
         Returns:
             loqus_versio(str)
         """
-        if loqusdb_id is None:
+        if loqusdb_id is None or loqusdb_id is "":
```
Comment (northwestwitch) [inline scout/server/extensions/loqus_extension.py:223]:
```suggestion
        if loqusdb_id is None or loqusdb_id == "":
```

File: scout/server/extensions/loqus_extension.py (hunk 1 above)
Comment (northwestwitch) [inline scout/server/extensions/loqus_extension.py:141]:
```suggestion
                    "-s",
```

**Review by northwestwitch — COMMENTED** [review]
There is some syntax error due to a typo

File: scout/server/extensions/loqus_extension.py (hunk 1 above)
Comment (mikaell) [reply scout/server/extensions/loqus_extension.py]: > There is some syntax error due to a typo
Fixed!

**Review by northwestwitch — APPROVED** [review]
I'm testing it for the last time on stage then we can merge I think, don't forget the CHANGELOG

**Comment by northwestwitch** [comment]
It's on stage now, looks like it's working fine!

**Comment by mikaell** [comment]
> I'm testing it for the last time on stage then we can merge I think, don't forget the CHANGELOG

Changelog updated. Will merge.

**Pull request merged**